index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
[OPTIONAL] 
- [ ] One of the good semantics to follow, os to add a `lang` attribute to the `html` tag to declare the language of this document.
```suggestion
<html lang="en">
    <head>
```